docs/releases/1.10.4.txt

```diff
@@ -12,3 +12,7 @@ Bugfixes
 * Quoted the Oracle test user's password in queries to fix the "ORA-00922:
   missing or invalid option" error when the password starts with a number or
   special character (:ticket:`27420`).
+
+* Striped trailing slashes for db_table option of model Meta to fix the "ORA-00911:
```

Comment:
Is this a regression? If not, it probably doesn't qualify for a backport to 1.10.